modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -193,6 +194,12 @@ public CPDefinition addCPDefinition(
 		CProduct cProduct = cProductLocalService.addCProduct(
 			externalReferenceCode, groupId, userId, new ServiceContext());
 
+		if (StringUtil.equalsIgnoreCase(
+				productTypeName, VirtualCPTypeConstants.NAME)) {
```

Comment:
Ehy @christian-chiappa I believe it is better not to change value of signature variables but rather do something like this

```suggestion
if (StringUtil.equalsIgnoreCase(productTypeName, VirtualCPTypeConstants.NAME)) {
    cpDefinition.setShippable(false);
} else {
    cpDefinition.setShippable(shippable);
}
```